cmd/eventlistenersink/main.go

```diff
@@ -39,26 +44,19 @@ const (
 )
 
 func main() {
-	ctx := signals.NewContext()
-
 	cfg := injection.ParseAndGetRESTConfigOrDie()
 
+	ctx := signals.NewContext()
+	ctx = injection.WithConfig(ctx, cfg)
+
 	dc := dynamic.NewForConfigOrDie(cfg)
 	dc = dynamicClientset.New(tekton.WithClient(dc))
 	ctx = context.WithValue(ctx, dynamicclient.Key{}, dc)
 
-	// Set up ctx with the set of things based on the
-	// dynamic client we've set up above.
-	ctx = injection.Dynamic.SetupDynamic(ctx)
-
 	sinkArgs, err := sink.GetArgs()
 	if err != nil {
 		log.Fatal(err.Error())
 	}
-	sinkClients, err := sink.ConfigureClients(ctx, cfg)
```

Comment:
Since this helper calls`cloudevent.Get(ctx)`  prior to the CE client being on the context means it would panic.

Reply:
Hence why I just created the clients below